docs/sources/developers/plugins/introduction-to-plugin-development/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Conceptual topics for plugin development
+title: Introduction to plugin development
+---
+
+This section contains topics related to the key concepts for Grafana plugin development.
+
+- [Backend plugins](backend/)
```

Comment:
This becomes a very bare page and again could benefit from some kind of summary for the links. I still think combine with create plugin generally.... but have this more as a quick start.

Reply:
This section is for conceptual topics, so a quick start doesn't fit our doc framework. But you are right to notice it's bare. I think this is a result of documentation gaps -- perhaps a lack of conceptual articles written so far, potential docs that we can write in the months ahead. Our first add should be to fill out the _index page with a general topic on the concepts behind plugin development.